src/app/Services/ProductService.ts

```diff
@@ -0,0 +1,58 @@
+import { Injectable } from "@angular/core";
+import { ProductItem } from "src/ProductItem";
+import { Observable } from "rxjs";
+import { backendURL } from "src/utils";
+import { HttpClient } from "@angular/common/http";
+import { ShoppingCartItem } from "src/ShoppingCardItem";
+
+@Injectable({providedIn:'root'})
+export class ProductService{
```

Comment:
**suggestion:**  strive to follow the angular styling guides for naming files https://angular.io/guide/styleguide#general-naming-guidelines. E.g.: In this case you name the file `ProductService` you should name it `products.service`